### ISSUE TITLE
Refactor bootstrap documentation URLs to /latest/ (when available)

### ISSUE THREAD

**Issue by PanderMusubi** [greyli/bootstrap-flask#176]
Refactor bootstrap documentation URLs to `/latest/` (when available). This is pending https://github.com/twbs/bootstrap/issues/28581#issuecomment-479360795 Upvoting the issue and/or comment can help.

This only is regarding documentation URLs for version 5, not version 4.

**Comment by greyli** [comment]
IMO, the current docs setup is OK.

**Comment by danielniccoli** [comment]
@greyli no, it is not. Google mostly refers to old version. If you use a link from your browser history, you mostly get to an old version because Bootstrap versioning moves quite fast. A `/latest/` url is very much needed. And it would be super simple to set up un addition to the current docs setup.

**Comment by greyli** [comment]
The `/latest/` URL is already supported: https://bootstrap-flask.readthedocs.io/en/latest/
